com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+{
+    "app-id": "com.kristianduske.TrenchBroom",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "trenchbroom.sh",
+    "rename-desktop-file": "trenchbroom.desktop",
+    "rename-icon": "trenchbroom",
+    "finish-args": [
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--filesystem=host",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "*.la",
+        "*.a"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1",
+            "DISPLAY": ":10"
+        }
+    },
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/gtk2/gtk2.json",
+        {
+            "name": "wxwidgets",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--with-opengl",
+                "--with-cxx=17",
+                "--with-gtk=2",
```

Comment:
I know that many people prefer gtk+2 for the wxWidgets applications because gtk+3 still has a lot of problems here. On the other hand, gtk+3 supports HiDPI and portals (especially File Chooser Portal, but this must be handled by the app itself). Anyway, have you tried to use gtk+3 version at least? Have you noticed any problems here?

Reply:
Hi, TrenchBroom author here ;-). Thanks for looking into this submission for us. We haven't tried gtk+3 when building wxWidgets. We have had our share problems with wxWidgets and are happy that our app now works on our 3 major platforms. We are also in the process of porting the entire app to QT, so we probably won't try gtk+3 either in the future.